src/Apps/Consign/Routes/ConsignmentInquiry/Components/ConsignmentInquiryFormAbandonEdit.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button, Flex, ModalDialog, Spacer, Text } from "@artsy/palette"
+import { useRouter } from "System/Router/useRouter"
+
+export const ConsignmentInquiryFormAbandonEditModal: React.FC<{
+  show: boolean
+  onClose: () => void
+}> = ({ show, onClose }) => {
+  const { router } = useRouter()
+
+  if (!show) {
+    return null
+  }
+
+  return (
+    <ModalDialog onClose={onClose} title="Leave without sending message?">
+      <Flex pb={2} flexDirection="column" alignItems={["stretch", "center"]}>
+        <Text>
+          Your message to the Sell with Artsy specialists will not been sent.
+        </Text>
+
+        <Spacer y={2} />
+
+        <Button
+          variant="primaryBlack"
+          onClick={() => {
+            router.go(-2)
+          }}
+          width="100%"
+          data-testid="get-in-touch-leave-button"
+        >
+          Leave Without Sending
+        </Button>
+
+        <Spacer y={1} />
+
+        <Button
+          variant="primaryWhite"
+          onClick={onClose}
+          width="100%"
+          data-testid="get-in-touch-continue-button"
+          borderColor="black100"
+        >
+          Continue Editing Message
+        </Button>
+      </Flex>
+      <Spacer y={4} />
```

Comment:
Avoid messing with the spacing in the dialogs. We've got a `footer` option if you need to stick a button at the bottom. Example: https://github.com/artsy/force/blob/main/src/Apps/Settings/Routes/Shipping/Components/SettingsShippingAddressForm.tsx#L182

Reply:
Actually: ![](https://static.damonzucconi.com/_capture/ZYDKDdeJ.png)

Just looks like that space is there by mistake. Should just remove it.